content/app-dev/create-provider/_index.en.md

```diff
@@ -5,10 +5,21 @@ weight: 5
 draft: false
 ---
 
-Creating a capability provider is one of the more involved tasks when working within the wasmCloud ecosystem. First, you need to decide if you're creating a new provider for an existing contract or if you're creating an entirely new contract.
+Creating a capability provider is one of the more involved tasks when working within the wasmCloud ecosystem. First, you need to decide if you're creating a new provider for an existing contract or if you're creating an entirely new contract. If you don't need to create a new contract, then you can skip to the second step in this guide.
 
-If you do not need to create a new contract, then you can skip to step 2 in this guide:
+### The Sample Use Case
 
-1. [Create a new contract](./new_contract)
-1. [Create the Rust capability provider](./rust)
-1. [Consume the new capability provider](./consuming) (create actors that use this provider)
+For the purposes of this guide, we're going to be creating a capability provider for a _payments service_ that accepts payments from customers. When composing applications with actors and capabilities, actors in an ecommerce setting may contain business logic that includes the need to accept payments from customers. There are a couple of ways we could solve this problem:
+
+* **Embed Vendor-Specific Knowledge** - In this scenario, an actor might make use of an _HTTP Client_ capability provider, and potentially use providers like a _key-value store_ and even an _event stream_. The actor could be performing a complex transactional dance of creating and retrieving tokens, accessing multiple data sources, and even making potentially long-lived synchronous calls to third parties over HTTP. Despite all our best efforts at isolating business logic in the self-contained unit of an actor, this actor would require recompilation and redeployment if any aspect of _how_ the company accepted payments were to change.
+* **Create a Capability Provider** - Hide the implementation details of accepting payments behind the contract of a capability provider, allowing for those details to be swapped at runtime without any impact on running actors.
+* **Create another Actor** - We could always defer the problem, and ask another actor to perform the payment processing for us. However, this just "kicks the can down the road" because that new actor would now have the same problem we have with the original one--_how to perform the actual payment processing?_
+

```

Comment:
Personally I would want to read this after the fact. I personally find that when I'm being taught how to do something I like to separate getting it done (ie creating the capability provider) with alternatives after the fact. I think it's an opportunity to teach best practices without explicitly doing so. I think you did write this way more so in your book.